src/metabase/notification/task/send.clj

```diff
@@ -110,7 +110,7 @@
                                                         :notification_subscription_id subscription-id
                                                         :cron_schedule                (:cron_schedule subscription)
                                                         :notification_ids             [notification-id]}}
-          (notification/send-notification! notification :notification/sync? true))
```

Comment:
this is a fix: we shouldn't send notification synchronously from quartz here, we want to send it async so it's queued properly.

This is a recent change from the alert PR, so it's only on master.